python/manylinux1/Dockerfile-x86_64

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-FROM quay.io/xhochy/arrow_manylinux1_x86_64_base:latest
+FROM quay.io/xhochy/arrow_manylinux1_x86_64_base:ARROW-3141
```

Comment:
Is this desired?

Reply:
Yes, this will pull the docker image from this PR as I have changed the base image. Once the PR is merged and the new image was uploaded, I will do a small maintenance PR that sets it back to `latest`.